views/marts/product/fct_active_users.view.lkml

```diff
@@ -137,34 +193,90 @@ view: fct_active_users {
-    label: "Desktop/Client WAU"
-    description: "WAU reported from Rudderstack"
+    label: "Desktop WAU"
+    description: "Desktop WAU reported by telemetry."
   }
 
   measure: total_weekly_desktop_active_users {
     type: sum
     sql: ${weekly_desktop_active_users} ;;
-    label: "Total Desktop/Client WAU"
+    label: "Total Desktop WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_webapp_active_users {
+    type: number
+    sql: ${TABLE}.weekly_webapp_active_users ;;
+    label: "Webapp WAU"
+    description: "Webapp WAU reported by telemetry."
+  }
+
+  measure: total_weekly_webapp_active_users {
+    type: sum
+    sql: ${weekly_webapp_active_users} ;;
+    label: "Total Webapp WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_client_desktop_active_users {
+    type: number
+    sql: ${TABLE}.weekly_client_desktop_active_users ;;
+    label: "Client Desktop WAU"
+    description: "Client WAU reported by Rudderstack telemetry."
+  }
+
+  measure: total_client_weekly_desktop_active_users {
+    type: sum
+    sql: ${weekly_client_desktop_active_users} ;;
+    label: "Total Client Desktop WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_legacy_desktop_active_users {
+    type: number
+    sql: ${TABLE}.weekly_legacy_desktop_active_users ;;
+    label: "Legacy Desktop WAU"
+    description: "Desktop WAU reported by Segment telemetry."
+  }
+
+  measure: total_weekly_legacy_desktop_active_users {
+    type: sum
+    sql: ${weekly_legacy_desktop_active_users} ;;
+    label: "Total Legacy Desktop WAU"
+    view_label: "  * Metrics. WAU"
+  }
+
+  dimension: weekly_client_webapp_active_users {
+    type: number
+    sql: ${TABLE}.weekly_client_webapp_active_users ;;
+    label: "Client Webapp WAU"
+    description: "Webapp WAU reported by Rudderstack telemetry."
+  }
+
+  measure: total_weekly_client_webapp_active_users {
+    type: sum
+    sql: ${weekly_client_webapp_active_users} ;;
+    label: "Total Client Webapp WAU"
     view_label: "  * Metrics. WAU"
   }
 
-  dimension: weekly_legacy_active_users {
+  dimension: weekly_legacy_webapp_active_users {
     type: number
-    sql: ${TABLE}.weekly_legacy_active_users ;;
-    label: "Legacy Desktop/Client WAU"
-    description: "WAU reported from Segment"
+    sql: ${TABLE}.weekly_legacy_webapp_active_users ;;
+    label: "Legacy Webapp WAU"
+    description: "WAU reported by Segment telemetry."
   }
 
-  measure: total_weekly_legacy_active_users {
+  measure: total_weekly_legacy_webapp_active_users {
     type: sum
-    sql: ${weekly_legacy_active_users} ;;
-    label: "Total Legacy Desktop/Client WAU"
+    sql: ${weekly_legacy_webapp_active_users} ;;
+    label: "Total Legacy Webapp WAU"
     view_label: "  * Metrics. WAU"
   }
 
   dimension: weekly_mobile_active_users {
     type: number
-    sql: ${TABLE}.weekly_mobile_active_users ;;
+    sql: ${TABLE}.monthly_mobile_active_users ;;
```

Comment:
monthly_mobile_active_users -> weekly_mobile_active_users

Reply:
this is updated, mobile as a whole has been dropped from the dbt model, so this has been removed.